docs/getting-started/gs-faqs.md

```diff
@@ -43,6 +44,16 @@ In addition, in almost all cases, UID2 doesn't store any values at all once the
 
 No. UID2 participants must not generate UID2s from Protected Health Information, as defined by the Health Insurance Portability and Accountability Act (HIPAA), even if they have obtained consent to engage in marketing with respect to such data.
 
+#### Should I use a Public Operator or a Private Operator?
+
+For most participants, <Link href="../ref-info/glossary-uid#gl-public-operator">Public Operator</Link> is the simplest solution. A Public Operator integration is a much easier option than creating your own <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>. However, having a Private Operator instance has many advantages. The best choice depends on your unique scenario and needs.
```

Comment:
_has advantages_  instead of _has many advantages_?

Reply:
@sunnywu unless you feel strongly about this I'd like to leave it. True there are not scores of advantages but "has advantages" is weak. There are significant advantages in my opinion, but it comes at a significant LOE. 
LMK if you feel strongly about this. If so I will make this edit.

Reply:
i don't have strong view - i just feel _many_ advantages seem to say it's far superior for all cases. But then you also wrote "The best choice depends on your unique scenario and needs." i'm ok either way

Reply:
Agree with Sunny, the current wording makes it sound like in general private operator is a better choice and masks the downsides. Maybe something like "having a Private Operator instance has some advantages, but adds extra complexities and costs"?

Reply:
Thx. Done. @sunnywu FYI.